### DISCUSSION TITLE
Addition of patch & dependencies

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1611] [Team Posts]
@xavierba,
- Could you add fix by sympa-community/sympa#842 ([patch](https://patch-diff.githubusercontent.com/raw/sympa-community/sympa/pull/842.patch
)) to 6.2.52 rpm?
- `perl(Socket6)` and `perl(IO::Socket::IP)` (or `perl(IO::Socket::INET6)` for EL6) are recommended to avoid some bugs.

**Comment by xavierba** [comment]
Hi @ikedas,

- I will add the patch for #842 later today.
- I have already added the IPv6 deps long ago, I was the one to hit and report the bug first ;-)
https://src.fedoraproject.org/rpms/sympa/c/ea13a590340b61c9c844974b80dfe9a1ae165459?branch=master
- If we could get to the bottom of #839, that would be awesome (sorry for not responding earlier, I have been sick the last 2 days, I'm back to it once I've finished a few morning tasks).

Regards,
Xavier

**Comment by ikedas** [comment]
- In addition, sympa-community/sympa#847 ([patch](https://github.com/sympa-community/sympa/pull/847.patch)).

On #839 please wait for a while.  I'll investigate it later.

**Comment by xavierba** [comment]
Hi @ikedas,

Both patches added (https://src.fedoraproject.org/rpms/sympa/c/97cbbcc8c6280cb7f74f8ab8ef216a9cbcc9f299?branch=master).

I will build for all supported Fedora and EPEL releases. Are you confident the fix is complete now ?

I hoped to include a fix for #839 at the same time, but I'll do it in another release.

Thanks,
Xavier

**Comment by ikedas** [comment]
@xavierba,
- ...in addition, sympa-community/sympa#848 and sympa-community/sympa#850 have to be applied.  

(If #839 was not reproduced, these four patches are all for 6.2.52 to work).
